src/testCasa.wtest

```diff
@@ -0,0 +1,33 @@
+import casa.*
+
+test "viveres y reparaciones" {
+	assert.equals(true,casaDePepeYJulian.tieneSufucientes())
+	assert.equals(false,casaDePepeYJulian.necesitaReparar())
+	casaDePepeYJulian.romperAlgo(1000)
+	assert.equals(true,casaDePepeYJulian.necesitaReparar())
+}

```

Comment:
para comprobar si algo es true o false se puede usar assert.that(casaDePepeYJulian.necesitaReparar()) para comprobar si es true o
assert.notThat(casaDePepeYJulian.necesitaReparar()) para comprobar si es false.

Los nombres de los test deben ser mas descriptvos con lo que se hace. Mencionar valores de inicio, que cosas se hacen y que valores finales tienen(esto para todos los tests)